src/App.tsx

```diff
@@ -289,8 +290,12 @@ const App = (): JSX.Element => {
           if (isLoading) setIsLoading(false);
         }, 3000);
         await loadPolkaBTC();
-        await loadFaucet();
-        keyring.loadAll({});
+        // Only load faucet on testnet
+        if (process.env.REACT_APP_BITCOIN_NETWORK !== 'mainnet') {
+          await loadFaucet();
+        }
+
+        // keyring.loadAll({});
```

Comment:
Minor: as with the import, unless we're planning to reintroduce this immediately it's a bit tidier to delete it rather than comment it out.